lib/v4/utils/run-jscodeshift.js

```diff
@@ -0,0 +1,23 @@
+const { join } = require('path');
+
+const jscodeshiftExecutable = require.resolve('.bin/jscodeshift');
+const execa = require('execa');
+
+/**
+ * @description Executes jscodeshift
+ *
+ * @param {string} path - the path where the transform should run
+ * @param {string} transform - the name of the transform file
+ * @param {object} options - execa options
+ */
+module.exports = (path, transform, options) => {
+  const result = execa.sync(
```

Comment:
I think it would be better to use the async version instead. Alex told me not to use execa sync on my templates PR